cmd/gokr-stat/stat.go

```diff
@@ -59,6 +60,7 @@ func terminalSize() (*window, error) {
 }
 
 func printStats() error {
+	var thermalFlag = flag.Bool("thermal", false, "system temperature sensors")
```

Comment:
Here and in webstat, can you make this a comma-separated list of module names?

var enabledModules = flag.String("modules", "cpu,disk,sys,net,mem", "comma-separated list of modules to show. known modules: cpu,disk,sys,net,mem,thermal")